config/services.yaml

```diff
@@ -4,6 +4,7 @@
 # Put parameters here that don't need to change on each machine where the app is deployed
 # https://symfony.com/doc/current/best_practices/configuration.html#application-related-configuration
 parameters:
+    locale: 'da'
```

Comment:
We should probably move this value to `.env` to make it easier to change the locale on deployment. In a short while this project will be a multilingual smash hit all over the OS2 world!